### ISSUE TITLE
Keyboard Accessibility

### ISSUE THREAD

**Issue by dominikbruechner** [saintplay/vue-swatches#28]
Hi Diego,
first of all - great work - I am looking forward to integrate you component into an educational open source project :) 

For accessibility reasons, accessing the colors by tab navigation would be great, on the one hand to open the popover, on the other hand to select colors within using TAB + Enter.
https://webaim.org/techniques/keyboard/

saludos 
Dominik

**Comment by saintplay** [comment]
Hi, I agree about accessibility need.

However, the next version of vue-swatches (v2) will only support `altText` #12  and `tab navigation`.
No popover accessibility at the moment. (Maybe v2.1)

The current workaround should be using one of the existing a11y popovers like [vue-a11y-dialog](https://github.com/morkro/vue-a11y-dialog) and `inline-mode`
